Support Gentoo-style autoconf wrapper.
Use existing automake version if present, else newest.
Add support for automake-1.11 (already in git for 2.28).

--- origsrc/gnome-common-2.26.0/macros2/gnome-autogen.sh	2009-06-08 01:06:11.804329300 -0500
+++ src/gnome-common-2.26.0/macros2/gnome-autogen.sh	2009-06-08 01:05:59.959651900 -0500
@@ -304,18 +304,21 @@ done
 #tell Mandrake autoconf wrapper we want autoconf 2.5x, not 2.13
 WANT_AUTOCONF_2_5=1
 export WANT_AUTOCONF_2_5
-version_check autoconf AUTOCONF 'autoconf2.50 autoconf autoconf-2.53' $REQUIRED_AUTOCONF_VERSION \
+#tell Gentoo autoconf wrapper we want autoconf 2.5x, not 2.13
+export WANT_AUTOCONF=2.5
+version_check autoconf AUTOCONF 'autoconf autoconf-2.5x' $REQUIRED_AUTOCONF_VERSION \
     "http://ftp.gnu.org/pub/gnu/autoconf/autoconf-$REQUIRED_AUTOCONF_VERSION.tar.gz"
 AUTOHEADER=`echo $AUTOCONF | sed s/autoconf/autoheader/`
 
 case $REQUIRED_AUTOMAKE_VERSION in
     1.4*) automake_progs="automake-1.4" ;;
-    1.5*) automake_progs="automake-1.10 automake-1.9 automake-1.8 automake-1.7 automake-1.6 automake-1.5" ;;
-    1.6*) automake_progs="automake-1.10 automake-1.9 automake-1.8 automake-1.7 automake-1.6" ;;
-    1.7*) automake_progs="automake-1.10 automake-1.9 automake-1.8 automake-1.7" ;;
-    1.8*) automake_progs="automake-1.10 automake-1.9 automake-1.8" ;;
-    1.9*) automake_progs="automake-1.10 automake-1.9" ;;
-    1.10*) automake_progs="automake-1.10" ;;
+    1.5*) automake_progs="automake-1.5 automake-1.11 automake-1.10 automake-1.9 automake-1.8 automake-1.7 automake-1.6" ;;
+    1.6*) automake_progs="automake-1.6 automake-1.11 automake-1.10 automake-1.9 automake-1.8 automake-1.7" ;;
+    1.7*) automake_progs="automake-1.7 automake-1.11 automake-1.10 automake-1.9 automake-1.8" ;;
+    1.8*) automake_progs="automake-1.8 automake-1.11 automake-1.10 automake-1.9" ;;
+    1.9*) automake_progs="automake-1.9 automake-1.11 automake-1.10" ;;
+    1.10*) automake_progs="automake-1.10 automake-1.11" ;;
+    1.11*) automake_progs="automake-1.11" ;;
 esac
 version_check automake AUTOMAKE "$automake_progs" $REQUIRED_AUTOMAKE_VERSION \
     "http://ftp.gnu.org/pub/gnu/automake/automake-$REQUIRED_AUTOMAKE_VERSION.tar.gz"
